internal/plugins/helm/v1/config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+// Config configures this plugin, and is saved in the project config file.
+type Config struct {
+	// ProjectName is the Helm operator's project name.
+	ProjectName string `json:"project"`
+}
```

Comment:
Since we will add it, then should we not add the check `hasPluginConfig` as well?

Reply:
For now at least, we should strive to have sane defaults for anything that's in the plugin config so that it is not required to be present.